### PR TITLE
fix: action panel backdrop opacity and fade in

### PR DESCRIPTION
**Pull request by pphminions**
## Description

Fix the opacity of the action panel  backdrop. This will fix the issue where when it is opening as a modal and the modal also has opacity.

## Motivation and Background Context

This is found while testing the action panel in the direct-web.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Screenshots (if appropriate):
Before the fix
Screen Shot 2019-06-27 at 10 52 20 am

After the fix
Screen Shot 2019-06-27 at 10 52 49 am

## Check-list:

- [x] I have read the [Contributing](CONTRIBUTING.md) document.
- [x] I've thought about and labelled my PR/commit message appropriately.
- [x] If this PR introduces breaking changes I've described the impact and migration path for existing applications.
- [x] CI is green (coverage, linting, tests).
- [x] I have updated the documentation accordingly.
- [x] I've two LGTMs/Approvals.
- [x] I've fixed or replied to all my code-review comments.
- [x] I've manually tested with a buddy.
- [x] I've squashed my commits into one.

### CODE REVIEW

File: src/components/adslot-ui/ActionPanel/index.jsx [hunk 1]
```diff
@@ -5,7 +5,8 @@ import Button from 'react-bootstrap/lib/Button';
 import './styles.scss';
 
 const ActionPanel = ({ title, className, size, onClose, children, actionButton, isModal, closeIcon }) => (
-  <div className={classNames('aui--action-panel-wrapper', { 'aui--action-panel-wrapper-backdrop': isModal })}>
+  <div className={classNames('aui--action-panel-wrapper', { 'aui--action-panel-modal-wrapper': isModal })}>
+    <div className={isModal ? 'aui--action-panel-wrapper-backdrop fade in' : 'hide'} />
```
Comment (pphminions) [inline src/components/adslot-ui/ActionPanel/index.jsx]: @veegandhi - the way I put `fade in` property might be incorrect.

File: src/components/adslot-ui/ActionPanel/styles.scss [hunk 2]
```diff
@@ -123,18 +124,32 @@
...
     left: 0;
     background-color: $color-gray-dark;
     width: 100%;
-    z-index: 1040;
-    display: flex;
-    justify-content: center;
+  }
+
+  &-wrapper-backdrop.fade.in {
     opacity: .9;
```
Comment (superpower1) [inline src/components/adslot-ui/ActionPanel/styles.scss]: Maybe just put this in `&-wrapper-backdrop.fade` underneath
```css
&-wrapper-backdrop.fade {
  opacity: 0;
  transition: opacity 30s linear;
  &.in {
    opacity: .9;
  }
}
```

File: src/components/adslot-ui/ActionPanel/styles.scss [hunk 3]
```diff
@@ -123,18 +124,31 @@
...
     width: 100%;
-    z-index: 1040;
-    display: flex;
-    justify-content: center;
-    opacity: .9;
+  }
+
+  &-wrapper-backdrop.fade {
+    opacity: 0;
+    transition: opacity 30s linear;
```
Comment (veegandhi) [inline src/components/adslot-ui/ActionPanel/styles.scss]:
```suggestion
    animation: fadein 0.2s;    
```

Comment (veegandhi) [reply src/components/adslot-ui/ActionPanel/styles.scss]: And then create a new  `src/styles/animation.scss` with
```scss
  @keyframes fadein {
    from {
      opacity: 0;
    }
    to {
      opacity: 0.9;
    }
  }
```

**Comment by pphminions** [comment]
@veegandhi - incorporated the review comments

File: src/components/adslot-ui/ActionPanel/styles.scss [hunk 4]
```diff
@@ -123,18 +125,31 @@
...
-    z-index: 1040;
-    display: flex;
-    justify-content: center;
-    opacity: .9;
+  }
+
+  &-wrapper-backdrop.fade {
+    opacity: 0;
+    animation: fadein 0.2s;
+    &.in {
```
Comment (veegandhi) [inline src/components/adslot-ui/ActionPanel/styles.scss]: this class is redundant, assign the `opacity:0.9` on l#136

File: src/components/adslot-ui/ActionPanel/index.jsx (hunk 1 above)
Comment (veegandhi) [reply src/components/adslot-ui/ActionPanel/index.jsx]: this class will now become `aui--action-panel-backdrop`

**Comment by pphminions** [comment]
@veegandhi - changed the code to fix the scrolling issue when the modal open

**Pull request merged**